pawar-2018/pawar2018
# Press releases

# What does this pull request do?

This adds a page for press releases and a link to that page in the footer. 

# How should the reviewer test this pull request?

Add the press releases custom post type, populate some press releases, and make sure they show up and look ok. (See details for the custom post type, etc, below). 

# Include the Basecamp todo for this PR, if there is one.

https://3.basecamp.com/3666950/buckets/2763013/todos/413603488

# Include any screenshots that will help explain this PR.

<img width="282" alt="screen shot 2017-04-06 at 10 48 03 pm" src="https://cloud.githubusercontent.com/assets/899/24785034/2c4c1d18-1b1c-11e7-9d44-10e07a8988f5.png">

<img width="347" alt="screen shot 2017-04-06 at 10 46 35 pm" src="https://cloud.githubusercontent.com/assets/899/24785036/320c9dae-1b1c-11e7-983b-3ac59608e268.png">

<img width="1084" alt="screen shot 2017-04-06 at 10 56 11 pm" src="https://cloud.githubusercontent.com/assets/899/24785048/49be6860-1b1c-11e7-8185-a00520c6978b.png">

# Any another context you want to provide?

This uses a custom post type, so the post type will need to be imported. Here's the post-type code (this includes events):

```
{"events":{"name":"events","label":"Events","singular_label":"Event","description":"","public":"true","publicly_queryable":"true","show_ui":"true","show_in_nav_menus":"true","show_in_rest":"false","rest_base":"","has_archive":"true","has_archive_string":"","exclude_from_search":"false","capability_type":"post","hierarchical":"false","rewrite":"true","rewrite_slug":"","rewrite_withfront":"true","query_var":"true","query_var_slug":"","menu_position":"","show_in_menu":"true","show_in_menu_string":"","menu_icon":"dashicons-calendar-alt","supports":["title"],"taxonomies":["post_tag"],"labels":{"menu_name":"","all_items":"","add_new":"","add_new_item":"","edit_item":"","new_item":"","view_item":"","view_items":"","search_items":"","not_found":"","not_found_in_trash":"","parent_item_colon":"","featured_image":"","set_featured_image":"","remove_featured_image":"","use_featured_image":"","archives":"","insert_into_item":"","uploaded_to_this_item":"","filter_items_list":"","items_list_navigation":"","items_list":"","attributes":""},"custom_supports":""},"press-releases":{"name":"press-releases","label":"Press Releases","singular_label":"Press Release","description":"","public":"true","publicly_queryable":"true","show_ui":"true","show_in_nav_menus":"true","show_in_rest":"false","rest_base":"","has_archive":"true","has_archive_string":"","exclude_from_search":"false","capability_type":"post","hierarchical":"false","rewrite":"true","rewrite_slug":"","rewrite_withfront":"true","query_var":"true","query_var_slug":"","menu_position":"","show_in_menu":"true","show_in_menu_string":"","menu_icon":"dashicons-calendar-alt","supports":["title"],"taxonomies":["post_tag"],"labels":{"menu_name":"","all_items":"","add_new":"","add_new_item":"","edit_item":"","new_item":"","view_item":"","view_items":"","search_items":"","not_found":"","not_found_in_trash":"","parent_item_colon":"","featured_image":"","set_featured_image":"","remove_featured_image":"","use_featured_image":"","archives":"","insert_into_item":"","uploaded_to_this_item":"","filter_items_list":"","items_list_navigation":"","items_list":"","attributes":""},"custom_supports":""}}
```

Here's a file with the field groups (it includes events field groups as well - if there's a way to export just one field group, I can do that. I could not find a way):

[pawardev.wordpress.2017-04-07.field-groups.xml.zip](https://github.com/pawar-2018/pawar2018/files/904503/pawardev.wordpress.2017-04-07.field-groups.xml.zip)


Here's a field with a bunch of lorem ipsum press releases, if that helps:

[pawardev.wordpress.2017-04-07-2.press-releases.xml.zip](https://github.com/pawar-2018/pawar2018/files/904505/pawardev.wordpress.2017-04-07-2.press-releases.xml.zip)

If there's a better way to distribute this configuration data, let me know.
